features/lwipstack/LWIPMemoryManager.h

```diff
@@ -160,6 +160,13 @@ class LWIPMemoryManager : public EMACMemoryManager {
      */
     virtual void set_len(emac_mem_buf_t *buf, uint32_t len);
 
+    /**
+     * Increases the reference count on the provided buffer
+     *
+     * @param buf      Memory buffer
+     */
+    void ref(emac_mem_buf_t *buf);
```

Comment:
This would need to be a proper `virtual` method in the base class with implementations for the test and lwIP stacks. Not needed yet without the bridge though, so suggest leaving it out for now.